tests/integration/golang/helpers/client.go

```diff
@@ -118,16 +118,21 @@ func (c *HttpClient) DoRequest(uri string, values ...any) error {
 	}
 
 	// 2. build path with namespace.
-	path := c.basePath
+	var path []string
```

Comment:
Can we just get rid of this now and `JoinPath` with whatever is in `c.namespace` (either something or empty string)?

Reply:
I tried to use the same approach for URL building which we use over the project.

Reply:
yes but in this case it's now redundant?

Reply:
this moment reverted from this PR right now.